apps/automation-v2/src/tlc/batch-liquidate.ts

```diff
@@ -160,14 +185,29 @@ export async function batchLiquidate(
   return taskSuccess();
 }
 
-const getTlcUsers = async (url: string) => {
-  const resp = await subgraphRequest<GetUserResponse>(url, {
-    query: `{
-      tlcUsers(where: {debt_gt: "0"}) {
-        id
-      }
-    }`,
-  });
-
-  return resp;
+const getTlcUsers = async (ctx: TaskContext, url: string, retries: number) => {
+  return backOff(
+    subgraphRequest<GetUserResponse>(
+      url,
+      {
+        query: `{
+        tlcUsers(block: {number: 18189826} where: {debt_gt: "0"}) {
+          id
+        }
+      }`,
+      },
+      ctx.logger
+    ),
+    {
+      numOfAttempts: retries,
+      retry: (e, attemptNumber) => {
+        if (e instanceof Error) {
+          ctx.logger.error(`subgraph retry ${attemptNumber} after rate limit`);
```

Comment:
> after rate limit
Do you know if it's from a rate limit here? Could be any error?

Reply:
Can probably remove the "after rate limit" part of the error string then? The error could be a rate limit or could be from any other error